youtube_dl/options.py

```diff
@@ -177,6 +177,11 @@ def _hide_login_info(opts):
         default=None, metavar='URL',
         help='Use the specified HTTP/HTTPS proxy. Pass in an empty string (--proxy "") for direct connection')
     network.add_option(
+        '--alternative-proxy', dest='alternative_proxy',
```

Comment:
Maybe rename it to `--extraction-proxy` and update the help message to something like `Proxy used for extracting the video info`, to make it clearer what it does.